src/views/Exchange/Exchange.js

```diff
@@ -173,8 +174,23 @@ export function getPositionQuery(tokens, nativeTokenAddress) {
 }
 
 export default function Exchange({ savedIsPnlInLeverage, setSavedIsPnlInLeverage, savedSlippageAmount, pendingTxns, setPendingTxns, savedShouldShowPositionLines, setSavedShouldShowPositionLines }) {
+  const [bannerHidden, setBannerHidden] = useState(false);
+
+  const hideBanner = (e) => {
+    const hiddenLimit = new Date(new Date().getTime()+(2*5*24*60*60*1000));
+    localStorage.setItem('bannerHidden', hiddenLimit)
+    setBannerHidden(true)
+  }
+
   useEffect(() => {
     window.scrollTo(0, 0)
+    const banner = localStorage.getItem('bannerHidden')
+    if (banner && new Date(banner) <= new Date()) {
```

Comment:
when i close banner it shows again after i reload page

i guess this condition should be `banner && new Date(banner) **>=** new Date()`

Reply:
also will be good to not show banner at all after trading competition is finished

Reply:
Fixed it.